ajax/libs/inheritance.js/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "inheritance.js",
+  "npmName": "inheritance-js",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "*.js",
+        "src/*.js"
```

Comment:
@PeterDaveHello I thought that the file hierarchy got flattened upon update. Is that not true? If not, then I can make some changes to the library so that the files will all appear in the same directory.